evm/pnpm-lock.yaml

```diff
@@ -0,0 +1,4089 @@
+lockfileVersion: '6.0'
```

Comment:
Surprising.

This set of projects uses Yarn. I'd not touch that decision on this PR. Can you remove this file from it?

Since you didn't touch package.json, I'm going to assume there's no version to change our minds about.